.github/workflows/ci.yaml

```diff
@@ -33,11 +33,12 @@ jobs:
       - name: Setup Conda Environment
         uses: conda-incubator/setup-miniconda@v3
         with:
-          miniforge-variant: Mambaforge
           miniforge-version: latest
-          use-mamba: true
           python-version: ${{ matrix.python-version }}
           activate-environment: test-environment
+          channels: conda-forge
+          conda-remove-defaults: true
```

Comment:
Oh, that's neat!

Reply:
Borrowed from Satpy (which I apparently also did :see_no_evil: :joy:).